WalletWasabi.Fluent/Global.cs

```diff
@@ -92,6 +93,10 @@ public Global(string dataDir, Config config, UiConfig uiConfig, WalletManager wa
 			Config.UseTor ? TorSettings.SocksEndpoint : null,
 			backendUriGetter: () => Config.GetBackendUri());
 
+		CoordinatorHttpClientFactory = new HttpClientFactory(
```

Comment:
CodeScene complains the code is getting more complex (even when the changes are pretty simple). Could you extract this logic to a method so we don't have the duplication? I think something like this:

```c#
	private HttpClientFactory BuildHttpClientFactory(Func<Uri> backendUriGetter) =>
		new (
			Config.UseTor ? TorSettings.SocksEndpoint : null,
			backendUriGetter);
```

Then this can be simplified as:

```c#
		HttpClientFactory = BuildHttpClientFactory(() => Config.GetBackendUri());
		CoordinatorHttpClientFactory = BuildHttpClientFactory(() => Config.GetCoordinatorUri());
```

Reply:
Gotcha, done